src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise(function(resolve, reject) {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+promise1.then(function() {
```

Comment:
```suggestion
promise1.then(() => {
```

Reply:
better practice to use arrow functions
it's faster and easier to code a little :)